### ISSUE TITLE
[Bug]: Error waiting for instance to finish resizing

### ISSUE THREAD

**Issue by benbro** [linode/terraform-provider-linode#545]
### Terraform Version

Terraform v1.0.7 on linux_amd64

### Linode Provider Version

version = "1.21.0"

### Effected Terraform Resources

linode_instance"

### Terraform Config Files

```terraform
# create
resource "linode_instance" "myinstance" {
  image = "linode/ubuntu20.04"
  label = "myinstance"
  region = "us-central"
  type = "g6-dedicated-8"
  root_pass = "123"
}

# resize
resource "linode_instance" "myinstance" {
  image = "linode/ubuntu20.04"
  label = "myinstance"
  region = "us-central"
  type = "g6-dedicated-16"
  root_pass = "123"
}
```

### Expected Behavior

Instance resize should succeed.

### Actual Behavior

Terraform show timeout error after 20 minutes although the resize succeeded.
```
Error: failed to change instance type: Error waiting for instance 111 to finish resizing: [002] Get "https://api.linode.com/v4/account/events?page=1": context deadline exceeded
```

### Steps to Reproduce

1. Create a g6-dedicated-8 instance
2. Resize the instance to g6-dedicated-16

**Comment by benbro** [comment]
Resizing from g6-dedicated-16 to g6-dedicated-8 returns immediately without waiting so resizing up/down is not symmetrical.

**Comment by LBGarber** [comment]
> The notification in the Linode cloud console says resizing from g6-dedicated-8 to g6-dedicated-16 took 22 minutes.

Ah got it! One thing to keep in mind is that `linode_resize` events are scheduled and may not start immediately but only report the duration of the actual resize. This seems to just be an abnormally long scheduling/updating process that exceeds the default `update` timeout. 

I'll see what we can do about adjusting the `update` timeout at the provider level, but in the meantime manually configuring the `update` field of the `timeouts` block should do the trick. 🙂

**Comment by LBGarber** [comment]
Hey there!

After some thought, I believe upping the `update` timeout at the provider level would increase the difficulty in identifying actual timeout issues and would not resolve the underlying issue with polling for scheduled events. With that being said, I recognize that occasional timeouts on long-running operations don't give a great user experience, so I'll add this to our internal tracker.

In the meantime, the issues with polling for multiple resize operations have been resolved and will be available in release `v1.25.0`.

Let us know if you run into anything else 😄
